lib/msal-common/src/auth/authority/Authority.ts

```diff
@@ -108,10 +108,8 @@ export abstract class Authority {
 
     public abstract async getOpenIdConfigurationAsync(): Promise<string>;
 
-    public async resolveEndpointsAsync(): Promise<ITenantDiscoveryResponse> {
+    public async resolveEndpointsAsync(): Promise<void> {
         const openIdConfigEndpoint = await this.getOpenIdConfigurationAsync();
         this.tenantDiscoveryResponse = await this.discoverEndpoints(openIdConfigEndpoint);
```

Comment:
what if these are not sucessful?

Reply:
Currently will throw an error and fail out. I think this is expected behavior, the only way this would fail is:
1. no network,
2. server unavailable,
3. endpoint configuration object has changed

Reply:
open for ideas on how to improve this.

Reply:
How do we distinguish between the above errors? Can we?

Reply:
resolved offline. We can distinguish by seeing where the error originates:
1. from fetch
2. from server
3. Our library